src/polyfills/stubs/intersection-observer-stub.js

```diff
@@ -181,39 +195,34 @@ export class IntersectionObserverStub {
     IntersectionObserverStub[UPGRADERS].push(this.upgrade_.bind(this));
   }
 
-  /**
-   * @return {?Element}
-   */
+  /** @return {?Element} */
   get root() {
     if (this.inst_) {
       return this.inst_.root;
     }
     return this.options_.root || null;
   }
 
-  /**
-   * @return {*}
-   */
+  /** @return {string} */
   get rootMargin() {
     if (this.inst_) {
       return this.inst_.rootMargin;
     }
-    return this.options_.rootMargin;
+    // The CC-provided IntersectionObserverInit type allows for rootMargin to be
+    // undefined, but we provide a default, so it's guaranteed to be a string
+    // here.
+    return /** @type {string} */ (this.options_.rootMargin);
   }
 
-  /**
-   * @return {*}
-   */
+  /** @return {!Array<number>} */
   get thresholds() {
     if (this.inst_) {
       return this.inst_.thresholds;
     }
     return [].concat(this.options_.threshold || 0);
   }
 
-  /**
-   * @return {undefined}
-   */
+  /** @function */
```

Comment:
This is just here to satisfy linting requiring jsdoc comments for every method

Reply:
Does the `@function` do anything? Can we remove it?